### PR TITLE
Add License, Copyright, and Info to Footer

### PR DESCRIPTION
**Pull request by liam-mulhall**
This PR adds a LICENSE.md, correct copyright info to the footer, and a message in the footer about images being released into the public domain.

### CODE REVIEW

File: src/views/minor/Footer.vue [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
+                Licensed under the
+                <a
+                    class="footer-link"
+                    href="https://opensource.org/licenses/MIT">
+                    MIT License</a
+                >.
+            </p>
+            <p>
+                By using this website, you agree any images you create are
+                released into the public domain.
```
Comment (katestange) [inline src/views/minor/Footer.vue:15]: Can we reword this as "By using this website, you agree that any images or media you create using the tools on this website are released by you into the public domain according to the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/)."

File: src/views/minor/Footer.vue [hunk 2]
```diff
@@ -1,20 +1,33 @@
...
         </div>
     </footer>
 </template>
 
 <script>
     const currentYear = new Date().getFullYear()
+    // prettier-ignore
     const copyrightText
-        = `Copyright © 2020-${currentYear} University of Colorado Boulder`
+    = `Copyright © 2020-${currentYear} Regents of University of Colorado`
```
Comment (katestange) [inline src/views/minor/Footer.vue:25]: "Regents of the University of Colorado" (I checked the university's own webpages for exact wording)

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+Copyright 2020-2022 Regents of University of Colorado
```
Comment (katestange) [inline LICENSE.md:1]: Regents of the University of Colorado

**Review by katestange — CHANGES_REQUESTED** [review]
I am double checking the public domain wording I proposed with the university lawyer.  Otherwise it looks great!

File: README.md [hunk 4]
```diff
@@ -1,5 +1,11 @@
 # Numberscope Frontend
 
+Copyright 2020-2022 Regents of University of Colorado.
```
Comment (katestange) [inline README.md:3]: Regents of the University of Colorado

File: src/views/minor/Footer.vue (hunk 2 above)
Comment (liam-mulhall) [reply src/views/minor/Footer.vue]: Should be fixed in https://github.com/numberscope/frontscope/commit/53aa6b1ae8f0d7baa5bddcdba7c69a9fb09fd8a9.

File: src/views/minor/Footer.vue (hunk 1 above)
Comment (liam-mulhall) [reply src/views/minor/Footer.vue]: Should be fixed in https://github.com/numberscope/frontscope/commit/53aa6b1ae8f0d7baa5bddcdba7c69a9fb09fd8a9. I omitted the word "license" as I wasn't sure if there's a legal distinction between a public domain "dedication" and a "license".

**Comment by liam-mulhall** [comment]
@katestange, should we add the note about images being released into the public domain in the README as well?

File: src/views/minor/Footer.vue [hunk 5]
```diff
@@ -1,20 +1,38 @@
...
+                Licensed under the
+                <a
+                    class="footer-link"
+                    href="https://opensource.org/licenses/MIT">
+                    MIT License</a
+                >.
+            </p>
+            <p>
+                By using this website, you agree any images you create are
+                released into the public domain according to
```
Comment (katestange) [inline src/views/minor/Footer.vue:15]: "By using this website, you agree that any images or media you create using the tools on this website are released by you into the public domain according to CC0."  Good catch about the word "license".  I still wanted to update the rest of the wording a bit, to include other media (we may have sound for example, or video), and to indicate that YOU are doing the releasing.

**Review by katestange — CHANGES_REQUESTED** [review]
About adding a public domain wording to the README.md.  I think the notice about public domain is directed at the website user, not the github user (i.e. someone planning to fork or install).  So, I guess I would leave it out of the README.md for now.  Glen may have an opinion?  Otherwise this review is just one more wording change.

File: src/views/minor/Footer.vue (hunk 5 above)
Comment (liam-mulhall) [reply src/views/minor/Footer.vue:15]: Whoops, I missed the bit about media. Adding it now...

Comment (liam-mulhall) [reply src/views/minor/Footer.vue]: Should be fixed in https://github.com/numberscope/frontscope/pull/119/commits/58cc120f6d5728f466183fd5ac5caff83116d4d2.

**Comment by katestange** [comment]
Thanks, it all looks great!  Maybe we should let Glen chime in before we merge?

**Comment by liam-mulhall** [comment]
Sounds good to me!

**Comment by gwhitney** [comment]
OK, have pushed the changes that create the version that received general approval on Slack. Given the fairly large changes in this round, @liam-mulhall thanks for reviewing one last time and merging. Hopefully you will be able to do something similar in the Vue3 PR once this goes into main and you rebase that. Let me know when that's all happened and the Vue3 PR is ready for review.

**Comment by liam-mulhall** [comment]
I'm happy with the changes you made, @gwhitney. I think @katestange is too, so I'll merge and then add the changes to my PR. I'll let you know when my PR is ready for review.
